src/test/java/root/receive/ReceiveMock.java

```diff
@@ -75,36 +75,36 @@ public void cleanup() throws Exception {
     @Test
     public void testPassedEntity() throws InterruptedException, ExecutionException, TimeoutException {
         ReadonlyMessage mockMessage = setupFakeResponse(ReceiveMessageTest.getMockMessageJson().toString(), false);
-        ReadonlyMessage readMessage = client.send("dummy").get(5, TimeUnit.SECONDS);
+        /*TODO: Broken, ReadonlyMessage readMessage = client.send("dummy").get(5, TimeUnit.SECONDS);
```

Comment:
Why are you commenting out unrelated tests? Please revert.

Reply:
They're fixed now, uncommented